src/app/containers/SwapHistory/index.tsx

```diff
@@ -222,32 +182,40 @@ export function SwapHistory() {
         {history.length > 0 && (
           <Pagination
             totalRecords={history.length}
-            pageLimit={6}
+            pageLimit={perPage}
             pageNeighbours={1}
             onChange={onPageChanged}
           />
         )}
       </div>
     </section>
   );
+};
+
+interface IAssetRowData {
+  status?: TxStatus;
+  timestamp: number;
+  transactionHash: string;
+  fromAmount: string;
+  toAmount: Nullable<string>;
 }
 
-interface AssetProps {
-  data: AssetRowData;
+interface IAssetProps {
+  data: IAssetRowData;
   itemFrom: AssetDetails;
   itemTo: AssetDetails;
 }
 
-function AssetRow({ data, itemFrom, itemTo }: AssetProps) {
+const AssetRow: React.FC<IAssetProps> = ({ data, itemFrom, itemTo }) => {
```

Comment:
Extract it to a separate component (create `components` folder in `SwapHistory`).